src/middlewared/middlewared/plugins/network.py

```diff
@@ -827,6 +827,9 @@ async def do_create(self, data):
             if interface_id:
                 await self.middleware.call('datastore.delete', 'network.interfaces', interface_id)
             raise
+        finally:
+            if await self.middleware.call('failover.licensed'):
+                await self.middleware.call('vrrpthread.set_non_crit_ifaces')
```

Comment:
Shouldn't we do this when the actual network configuration is applied instead?

Reply:
You mean during `interface.sync`?

Reply:
Yes (if that's what changes networking on HA)